Gemfile

```diff
@@ -0,0 +1,5 @@
+source 'https://rubygems.org'
+
+gemspec
+
+gem 'rspec'
```

Comment:
Including the `Gemfile` allows us now to run `bundle`. `#gemspec` is a built-in method that allows you to use the `payrix.gemspec` dependencies mentioned there as one source for the gems to download.

I have added `rspec` for testing.